protobufs/livekit_agent.proto

```diff
@@ -78,6 +78,7 @@ message ServerMessage {
 enum JobType {
     JT_ROOM = 0;
     JT_PUBLISHER = 1;
+    JT_CUSTOM = 2;
```

Comment:
maybe JT_MANUAL? Custom implies it still happens automatically but with customer conditions.